Agents/ResultedConsumptionCalculationAgent/README.md

```diff
@@ -0,0 +1,216 @@
+## Description
+This `Resulted (Energy) Consumption Calculation` agent is designed to calculate the resulted electricity and gas consumption based on current energy consumptions, COP, with several assumptions and indecies, and instantiated in the [The World Avatar] KG according to the [OntoRegionalAnalysis] ontology. 
+
+The agent is implemented as Docker container to be deployed to a Docker stack spun up by the [Stack Manager]. It is recommended to use `VS Code` to develop/deploy the agent. Hence, a few of the details below are VS Code specific.
+
+This agent should be implemented with other agents using Derived Information Framework. This agent works with [LSOAInputAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/LSOAInputAgent), [CopCalculationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/CopCalculationAgent), [UtilityCostCalculationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/UtilityCostCalculationAgent) and [InequalityIndexCalculationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/InequalityIndexCalculationAgent), to support the analysis of the impact of heat pumps in the UK (for details refer to **[Preprint 323](https://como.ceb.cam.ac.uk/preprints/323/)** with **[Preprint 281](https://como.ceb.cam.ac.uk/preprints/281/)**). You may refer to the following graph to find what is the role of this agent in these calculations:
+
+![Agent framework](https://i.imgur.com/vSBvBoJ.jpeg)
+
+This agent uses Derived Infomation Framework, details provided below:
+
+
+### Basic
+The underlying equation of the calculations in this agent is:
+
+
+$$
+Change of gas consumption = uptake × gas consumption × propotion of heating
+$$
+
+$$
+Change of electricityconsumption  = \frac{boiler efficiency × change of gas}{COP}
+$$
+
+$$
+Resulted Gas consumption = Gas consumption - Change of gas consumption
+$$
+
+$$
+Resulted Electricity consumption = Electricity consumption + Change of electricity consumption
+$$
+
+**Arguments**:
+
+<u>Uptake</u>: a index which quantify the extent of the deployment of heat pump by measureing the gas being replaced by using heat pump comparing to conventional fossil fule heating system.
```

Comment:
Could you put the arguments on separate lines? The current text is very difficult to read. Thank you.